mitodl/ocw-studio
# set page title

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #724

#### What's this PR do?

This PR adds page titles to the frontend app. I created a component called `DocumentTitle` which basically has a side-effect of setting the page title, and added that throughout the app.

#### How should this be manually tested?

Go through the app and make sure there are page titles where they should be! Check the issue for the proposed titles.

#### Screenshots (if appropriate)


![Screen Shot 2021-11-02 at 4 36 59 PM](https://user-images.githubusercontent.com/6207644/139948020-3938c7a6-f741-4192-9835-adca27902b73.png)
